operator/pkg/manifest/installer.go

```diff
@@ -613,12 +618,33 @@ func objectsNotInLists(objects object.K8sObjects, lists ...object.K8sObjects) ob
 	return ret
 }
 
-func waitForCRDs(objects object.K8sObjects, dryRun bool) error {
+func canSkipCrdWait(applyOut string) bool {
```

Comment:
Nit: initialisms https://github.com/golang/go/wiki/CodeReviewComments#initialisms
canSkipCRDWait

Reply:
Thanks, added this to cleanup task for 1.6: https://github.com/istio/istio/issues/21435